README.md

```diff
@@ -15,13 +15,13 @@
   </a>
   <br />
   <br />
-  <a href="https://docs.geldata.com/get-started/quickstart">Quickstart</a>
+  <a href="https://docs.geldata.com/learn/quickstart/overview/nextjs">Quickstart</a>
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
   <a href="https://www.geldata.com">Website</a>
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://docs.geldata.com/libraries/js">Docs</a>
+  <a href="https://docs.geldata.com/reference/clients/js#gel-js-intro">Docs</a>
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://discord.gg/umUueND6ag">Discord</a>
+  <a href="https://discord.com/channels/841451783728529451/849374705210490900">Discord</a>
```

Comment:
This needs to be an invite code, not a link to a channel. Did that code not work?

Reply:
My bad, the original invite link seems OK. I think I encountered a broken discord link in a different spot.